packages/next/src/server/lib/clone-response.ts

```diff
@@ -1,3 +1,22 @@
+// @ts-ignore https://nodejs.org/api/stream.html#streamreadableisdisturbedstream
+import { isDisturbed, isErrored } from 'node:stream'
```

Comment:
If I read the docs correctly, these are statics on `Readable`. Can't use `node:` either since it needs to work in Edge runtimes.
```suggestion
import { Readable } from 'stream'
```

Reply:
Thank you, let me update this

Reply:
<img width="969" height="263" alt="image" src="https://github.com/user-attachments/assets/8ffe2d5e-7150-4b72-9120-f3862d959996" />

I'm still not able to get the `isDisturbed` or `isErrored` checks working type-wise. However, I'm also not certain they're needed at all.

> The stream you are trying to cancel is not a [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream), or it is locked.

https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/cancel#exceptions

Additionally, we're also catching the error, if one were to occur.